src/components/orgSelector.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from "react";
+import { useAllOrgs } from "dcs-react-hooks";
+import Autocomplete from "@mui/material/Autocomplete";
+import TextField from "@mui/material/TextField";
+
+export default function Organization() {
+
+  const { state, actions } = useAllOrgs({ lang: "en", page: 1, limit: 5 });
```

Comment:
Looks like only having `limit: 5` doesn't allow us to see and use `unfoldingWord` in the dropdown. Can you increase this to 500? That should give us plenty of room to add new organizations to the server before needing to increase the limit again.